### PR TITLE
feature: 完成全局变量 集群分组选择器（3.6.X）

### PR DESCRIPTION
**Pull request by Atropos1999**

### CODE REVIEW

**Comment by tencent-adm** [comment]
[CLA assistant check](https://cla.opensource.tencent.com/Tencent/bk-sops?pullRequest=2764) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.tencent.com/Tencent/bk-sops?pullRequest=2764) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.tencent.com/check/Tencent/bk-sops?pullRequest=2764) it.</sub>

File: pipeline_plugins/variables/collections/sites/open/docs/cmdb/var_set_group_selector.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+* 返回格式
+```
+{
+	"bk_alias_name": [],
+	"bk_capacity": [],
+	"bk_category": [],
+	"bk_chn_name": [],
+	"bk_customer": [],
+	"bk_enable_relate_webplat": [],
```
Comment (homholueng) [inline pipeline_plugins/variables/collections/sites/open/docs/cmdb/var_set_group_selector.md:42]: 这里返回格式的示意按照外部版的来写

File: pipeline_plugins/variables/collections/sites/open/docs/cmdb/var_set_group_selector.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+### 输入参数
+
+* `set_group_selector`: 集群分组ID 
+
+### 返回参数及格式
+
+* `bk_set_id` 集群ID列表
+* `bk_set_name` 集群名称列表
+* `flat__set_id` 字符串格式的集群名称
+* `flat__set_name` 字符串格式的集群ID
```
Comment (homholueng) [inline pipeline_plugins/variables/collections/sites/open/docs/cmdb/var_set_group_selector.md:31]: 分隔符要说明

File: pipeline_plugins/variables/collections/sites/open/cmdb/var_set_group_selector.py [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+from gcloud.conf import settings
+from django.utils.translation import ugettext_lazy as _
+
+from gcloud.utils.cmdb import batch_request
+from pipeline.core.data.var import LazyVariable
+
+logger = logging.getLogger("root")
+get_client_by_user = settings.ESB_GET_CLIENT_BY_USER
+
+set_field = [
```
Comment (homholueng) [inline pipeline_plugins/variables/collections/sites/open/cmdb/var_set_group_selector.py:25]: 这里也不能写死，需要拉取 cmdb 当前业务下集群模型有哪些属性，再请求这些属性

**Review by homholueng — CHANGES_REQUESTED** [review]

File: pipeline_plugins/variables/collections/sites/open/cmdb/var_set_group_selector.py [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+
+    def get_value(self):
+        """
+        获取该变量中对应属性值
+        """
+        operator = self.pipeline_data.get("executor", "")
+        bk_biz_id = int(self.pipeline_data.get("biz_cc_id", 0))
+        bk_group_id = self.value
+        set_module_info = cc_execute_dynamic_group(operator, bk_biz_id, bk_group_id)
+        set_field = get_set_property(operator)
```
Comment (homholueng) [inline pipeline_plugins/variables/collections/sites/open/cmdb/var_set_group_selector.py:98]: cc_execute_dynamic_group 调用过一次 get_set_property，这里又调用了一次，可以在 cc_execute_dynamic_group 之前执行一次 get_set_property 获取集群属性就好了吧

**Review by homholueng — CHANGES_REQUESTED** [review]

**Review by homholueng — APPROVED** [review]

**Pull request merged**